crates/stackable-webhook/src/constants.rs

```diff
@@ -5,8 +5,10 @@ use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 /// The default HTTPS port `8443`
 pub const DEFAULT_HTTPS_PORT: u16 = 8443;
 
-/// The default IP address `127.0.0.1` the webhook server binds to.
-pub const DEFAULT_IP_ADDRESS: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
+/// The default IP address [`Ipv4Addr::UNSPECIFIED`] (`0.0.0.0`) the webhook server binds to,
+/// which represents binding on all network interfaces.
+pub const DEFAULT_LISTEN_ADDRESS: IpAddr = IpAddr::V4(Ipv4Addr::UNSPECIFIED);
 
-/// The default socket address `127.0.0.1:8443` the webhook server vinds to.
-pub const DEFAULT_SOCKET_ADDR: SocketAddr = SocketAddr::new(DEFAULT_IP_ADDRESS, DEFAULT_HTTPS_PORT);
+/// The default socket address `0.0.0.0:8443` the webhook server binds to.
+pub const DEFAULT_SOCKET_ADDR: SocketAddr =
```

Comment:
suggestion: While we are renaming constants, I would like this constant to be non-abbreviated.

```suggestion
pub const DEFAULT_SOCKET_ADDRESS: SocketAddr =
```

Reply:
0c72fe8